src/utils/config-helper.ts

```diff
@@ -55,10 +57,6 @@ export function fetchConfig (): Promise<any> {
       const addressCompleteKey = response.data['ADDRESS_COMPLETE_KEY'];
       (<any>window).addressCompleteKey = addressCompleteKey
       console.info('Set Address Complete Key.')
-
```

Comment:
This was a recent fix that you're undoing. Please revert this change.

Reply:
poor rebase on my part, corrected in latest commit